src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAccountPayeController.cs

```diff
@@ -21,18 +20,18 @@ public class EmployerAccountPayeController : BaseController
         private const int AddPayeUsingAorn = 2;
 
         private readonly EmployerAccountPayeOrchestrator _employerAccountPayeOrchestrator;
+        private readonly UserAornLockOrchestrator _userAornLockOrchestrator;
```

Comment:
Looking at the number of methods this is used in I'm not convinced it should be added as a dependency as most methods don't need it.  If you look at how many places have had to be changed to simply pass a new mock object it highlights the issue.  Could we do it using an attribute?